Neos.Cache/Classes/Backend/SimpleFileBackend.php

```diff
@@ -19,7 +19,6 @@
 use Neos\Cache\Frontend\PhpFrontend;
 use Neos\Cache\Frontend\FrontendInterface;
 use Neos\Utility\Exception\FilesException;
-use Neos\Utility\Lock\Lock;
```

Comment:
Do we still use Neos\Utility\Lock\Lock somewhere or can we remove the thing then?